app_dart/lib/src/request_handlers/push_build_status_to_github.dart

```diff
@@ -53,10 +53,12 @@ class PushBuildStatusToGithub extends ApiRequestHandler<Body> {
   Future<void> _updatePRs(RepositorySlug slug, String status, DatastoreService datastore) async {
     final GitHub github = await config.createGitHubClient(slug: slug);
     final List<GithubBuildStatusUpdate> updates = <GithubBuildStatusUpdate>[];
-    await for (PullRequest pr in github.pullRequests.list(
-      slug,
-      base: Config.defaultBranch(slug),
```

Comment:
This wasn't working, so I moved it into the for loop (the same way gold skips release branches)